packages/libs/react-ui/src/components/Input/Input.stories.tsx

```diff
@@ -40,6 +41,15 @@ const meta: Meta<
         type: 'text',
       },
     },
+    leadingTextWidth: {
```

Comment:
In the story of Inputgroup I also see leadingTextwidth as a prop. But does not do anything.
![image](https://github.com/kadena-community/kadena.js/assets/4015521/ce769889-d9a7-454e-9498-226d45379922)

Reply:
Should be fixed with the new ellipsis strategy